recipes/omicverse/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "omicverse" %}
+{% set version = "1.4.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3cd8e2d4a93d813316651db1d1dfb6cdab7714b61843f7229e18c136336fe2e3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+
+requirements:
+  host:
+    - flit >=3.2,<4
+    - python >=3.8,<=3.9
+    - pip
+    
+  run:
+    - python >=3.8
+    - numpy ==1.23.5
+    - pandas ==1.5.3
```

Comment:
Are those strict pins really necessary?

Reply:
Thanks for your comment, we set numpy >=1.22,<1.24, pandas <=1.5.3  could run successfully in new pr too. 
- 1.5.3 is the last version of `pandas` 1.0, the next version is 2.0 and a large number of api's have been rewritten leaving many of the associated dependencies unsupported. 
- `numpy` 1.24 also changed some of the underlying api names accordingly, so the original dependency limit was lower than 1.23 and higher than 1.22. To ensure the smooth operation of the omicverse, we restricted the version of numpy.